File: sea-shell-service-immutable/src/main/java/org/jesperancinha/shell/webflux/immutable/data/SeaShellPersonDto.java
package org.jesperancinha.shell.webflux.immutable.data;

import lombok.Builder;

public record SeaShellPersonDto(
        String name,
        String activity,
        SeaShellCostumeDto costumeDto,
        SeaShellAccountDto accountDto,
        Long costumeId,
        String accountId
) {
    @Builder
    public SeaShellPersonDto {
    }
}

File: sea-shell-service-immutable/src/main/kotlin/org/jesperancinha/shell/webflux/immutable/data/SeaShellPersonDto.kt
package org.jesperancinha.shell.webflux.immutable.data

data class SeaShellPersonDto(
    val name: String,
    val activity: String? = null,
    val costumeDto: SeaShellCostumeDto,
    val accountDto: SeaShellAccountDto,
    val costumeId: Long,
    val accountId: String
)